lib/galaxy/tool_util/parser/interface.py

```diff
@@ -296,6 +296,13 @@ def parse_creator(self):
         """
         return []
 
+    def parse_funding(self):
```

Comment:
One probably wants to have this is the yaml parser as well?

Reply:
Good point! Didn't catch the support for that.